agent/bench-scripts/gold/test-benchmark-clis/test-CL.txt

```diff
@@ -1841,19 +1691,13 @@ Usage: pbench-user-benchmark [options] -- <script to run>
 --- Finished test-CL test-benchmark-clis (status=0)
 +++ pbench tree state
 /var/tmp/pbench-test-bench/pbench-agent
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log
 /var/tmp/pbench-test-bench/pbench-agent/tmp
 /var/tmp/pbench-test-bench/pbench-agent/tools-default
 /var/tmp/pbench-test-bench/pbench-agent/tools-default/mpstat
 /var/tmp/pbench-test-bench/pbench-agent/tools-default/sar
 --- pbench tree state
 +++ pbench.log file contents
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] This script (pbench-moongen) is deprecated and will be removed in a future version of pbench (v0.61).
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] Please switch ASAP to pbench-trafficgen, which provides MoonGen and TRex based traffic generators.
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] This script (pbench-moongen) is deprecated and will be removed in a future version of pbench (v0.61).
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] Please switch ASAP to pbench-trafficgen, which provides MoonGen and TRex based traffic generators.
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] This script (pbench-moongen) is deprecated and will be removed in a future version of pbench (v0.61).
-/var/tmp/pbench-test-bench/pbench-agent/pbench.log:[warn][1900-01-01T00:00:00.000000] Please switch ASAP to pbench-trafficgen, which provides MoonGen and TRex based traffic generators.
+grep: /var/tmp/pbench-test-bench/pbench-agent/pbench.log: No such file or directory
```

Comment:
This seems peculiar, where did this come from?

Reply:
I think that the deprecation notice that you added in PR #1190 ended up in that file (and was the only thing that did so). I'll see if I can rid of the error message.

Reply:
If we merge #1254, we'll pick up the change that removes all of those useless "pbench.log: No such file or directory" error messages in the gold files.

Reply:
Sounds good to me.